src/main/java/org/prebid/server/auction/AuctionRequestFactory.java

```diff
@@ -494,6 +501,21 @@ private static JsonNode populatePriceGranularity(JsonNode priceGranularityNode,
         return result;
     }
 
+    /**
+     * Returns populated {@link ExtRequestPrebidCache} or null if no changes were applied.
+     */
+    private ExtRequestPrebidCache cacheOrNull(ExtRequestPrebid prebid) {
+        final ExtRequestPrebidCache cache = prebid != null ? prebid.getCache() : null;
+        final Boolean cacheWinningOnly = cache != null ? cache.getWinningonly() : null;
+        if (cacheWinningOnly == null) {
```

Comment:
i'd like to add another condition: `&& shouldCacheOnlyWinningBids`.
So, we'll produce updated `ext.prebid.cache` object if reasonable.

PS. i don't think we need to add `"winningonly": false` in request if it was not passed.